caddy/setup/browse.go

```diff
@@ -280,7 +300,7 @@ footer {
 								{{else}}
 								<svg width="1.5em" height="1em" version="1.1" viewBox="0 0 26.604381 29.144726"><use xlink:href="#file"></use></svg>
 								{{end}}
-								{{.Name}}
+								<div class="name">{{.Name}}</div>
```

Comment:
The parent div also has the class of "name" - but I don't think it's needed, actually, we can probably remove that. (line 297)